public/service_portal/v0.1.0/openapi.json

```diff
@@ -190,23 +190,10 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "Edited portfolio object",
-                        "content": {
-                            "application/json": {
-                                "schema": {
-                                    "$ref": "#/components/schemas/Portfolio"
-                                }
-                            }
-                        }
+                        "description": "Portfolio Deleted"
                     },
-                    "401": {
-                        "$ref": "#/components/responses/Unauthorized"
-                    },
-                    "403": {
-                        "$ref": "#/components/responses/Forbidden"
-                    },
-                    "422": {
-                        "$ref": "#/components/responses/InvalidEntity"
+                    "404": {
+                        "description": "Portfolio Not Found"
```

Comment:
@lindgrenj6 - Should we also have a 204 for `:no_content` as reported back from the controller here: https://github.com/ManageIQ/service_portal-api/blob/master/app/controllers/api/v0/admins_controller.rb#L18-L20

Reply:
Ah yep you're right. When a delete happens successfully it's a 204 rather than 200. I'll update it.